src/common/utils/registry/repository.go

```diff
@@ -277,9 +277,9 @@ func (r *Repository) MountBlob(digest, from string) error {
 	if err != nil {
 		return err
 	}
+	defer resp.Body.Close()
```

Comment:
with this the connection will be closed if the status code is 200?

Reply:
This maybe not a connection leak. But we should call `resp.Body.Close()` regardless of the status code.